add-ons/ansible-service-broker/ansible-service-broker.addon

```diff
@@ -1,12 +1,14 @@
 # Name: ansible-service-broker
 # Description: Deploys the Ansible Service Broker
-# Required-vars:
+# Required-vars: BROKER_REPO_TAG, APBTOOLS_REPO_TAG, DOCKERHUB_ORG
```

Comment:
Typo here, It should be `Required-Vars`. See https://docs.openshift.org/latest/minishift/using/addons.html#addon-dynamic-variables